src/runtime_config/constitution.js

```diff
@@ -104,8 +88,19 @@ const actions = new Map([
       function (args) {
         return true;
       },
-      function (args, tx) {
-        return true;
+      function (args) {}
+    ),
+  ],
+  [
+    "remove_user",
+    new Action(
+      function (args) {
+        return typeof args.user_id === "string";
```

Comment:
We should probably add a function to check that a giver id is valid (i.e. hex-encoded string of SHA-256).

Reply:
Yes, let's refine the actions as we implement them, this is just a testcase for the apply path, not necessarily the final remove_user action.